### PR TITLE
WEB-Email-MXplan-auto-replies-SK1378

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-updated: 2021-04-20
+title: 'MXplan - Créer une réponse automatique sur une adresse e-mail'
+excerpt: 'Découvrez comment mettre en place une réponse automatique sur une adresse e-mail'
+updated: 2024-05-24
 ---
 
 ## Objectif
 
-En cas d'absence à votre bureau, vous pouvez mettre en place un répondeur e-mail qui laissera un message aux interlocuteurs souhaitant vous contacter par e-mail.
+Lorsque vous êtes absent et que vous n'êtes pas en mesure de consulterez votre adresse e-mail, vous pouvez mettre en place une réponse automatique (ou répondeur) qui transmettra un e-mail aux interlocuteurs qui vous envoie un e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
Lorsque vous êtes absent et que vous n'êtes pas en mesure de consulterez votre adresse e-mail, vous pouvez mettre en place une réponse automatique (ou répondeur) qui transmettra un e-mail aux interlocuteurs qui vous envoie un e-mail.
```
```suggestion
Lorsque vous êtes absent et que vous n'êtes pas en mesure de consulter votre adresse e-mail, vous pouvez mettre en place une réponse automatique (ou répondeur) qui transmettra un e-mail aux interlocuteurs qui vous envoient un e-mail.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
 
-**Découvrez comment mettre en place un répondeur e-mail.**
+**Découvrez comment mettre en place une réponse automatique sur une adresse e-mail.**
 
 ## Prérequis
 
-- Disposer d'une offre MX Plan. Celle-ci est disponible via : une offre d’[hébergement web](https://www.ovhcloud.com/fr/web-hosting/){.external}, l'[hébergement gratuit 100M](https://www.ovhcloud.com/fr/domains/free-web-hosting/){.external} compris avec un nom de domaine (activé au préalable) ou l'offre MX Plan commandée séparément.
-- Être connecté à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}.
+- Disposer d'une offre MX Plan. Celle-ci est disponible via : une offre d’[hébergement web](/links/web/hosting), l'[hébergement gratuit 100M](/links/web/domains-free-hosting) compris avec un nom de domaine (activé au préalable) ou l'offre MX Plan commandé séparément.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
- Disposer d'une offre MX Plan. Celle-ci est disponible via : une offre d’[hébergement web](/links/web/hosting), l'[hébergement gratuit 100M](/links/web/domains-free-hosting) compris avec un nom de domaine (activé au préalable) ou l'offre MX Plan commandée séparément.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 3]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
 
 - `Type de répondeur`:
 
-« Associé à une boite e-mail » : à utiliser si cela concerne une adresse e-mail existante sur votre offre e-mail.
-« Libre » : à utiliser dans le cas d'un alias. Il n'est donc pas lié à une adresse existante.
-
-- `Boîte email` ou `Nom du répondeur`: l'adresse e-mail ou l'alias concerné par le répondeur.
-
-- `Durée du répondeur `:
+**Associé à une boite e-mail**: à utiliser si cela concerne une adresse e-mail existante sur votre offre e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
**Associé à une boite e-mail** : à utiliser si cela concerne une adresse e-mail existante sur votre offre e-mail.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 4]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
+**Libre**: à utiliser dans le cas d'un alias. Il n'est donc pas lié à une adresse existante.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
**Libre** : à utiliser dans le cas d'un alias. Il n'est donc pas lié à une adresse existante.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 5]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
 
-« Temporaire » : définissez une date de début et de fin à prendre en compte pour le fonctionnement de votre répondeur (utile si vous partez en congés par exemple).
-« Permanent » : le répondeur fonctionnera tant que vous ne l'aurez pas désactivé.
-
-- `Envoyer une copie ` ou `Garder les messages sur le serveur `: permet de renvoyer les messages reçus pendant votre absence vers l'adresse de votre choix ou de les conserver sur l'adresse e-mail.
+- `Boîte email` ou `Nom du répondeur`: l'adresse e-mail ou l'alias concerné par la réponse automatique.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md:41]:
```suggestion
- `Boîte email` ou `Nom du répondeur` : l'adresse e-mail ou l'alias concerné par la réponse automatique.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 6]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
+- `Durée du répondeur`:
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md:42]:
```suggestion
- `Durée du répondeur` :
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 7]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
+    - **Temporaire** : définissez une date de début et de fin à prendre en compte pour le fonctionnement de votre réponse automatique (utile si vous partez en congé par exemple).
+    - **Permanent** : la réponse automatique fonctionnera tant que vous ne l'aurez pas désactivé.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
    - **Permanent** : la réponse automatique fonctionnera tant que vous ne l'aurez pas désactivée.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 8]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
+- `Envoyer une copie` ou `Garder les messages sur le serveur`: permets de renvoyer les messages reçus pendant votre absence vers l'adresse de votre choix ou de les conserver sur l'adresse e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md:45]:
```suggestion
- `Envoyer une copie` ou `Garder les messages sur le serveur` : permet de renvoyer les messages reçus pendant votre absence vers l'adresse de votre choix ou de les conserver sur l'adresse e-mail.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 9]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
 
 > [!warning]
+>
 > Si vous décochez cette case, les messages reçus pendant votre absence seront automatiquement supprimés.
 
-- `Adresse en copie ` (Seulement en mode libre) : dans le cas d'un alias, sélectionnez l'adresse e-mail qui recevra les e-mails envoyés vers l'alias.
+- `Adresse en copie` (seulement en mode libre) : dans le cas d'un alias, sélectionnez l'adresse e-mail qui recevra les e-mails envoyés vers l'alias.
+- `Message`: Il s'agit du message que vos correspondants recevront lorsqu'ils vous enverront un e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
- `Message`: il s'agit du message que vos correspondants recevront lorsqu'ils vous enverront un e-mail.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 10]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
 
-- `Message `: Il s'agit du message que vos correspondants recevront lorsqu'ils vous enverront un e-mail.
+Cliquez ensuite sur `Valider`{.action} pour terminer la configuration de votre réponse automatique soit mise en place.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
Cliquez ensuite sur `Valider`{.action} pour terminer la configuration de votre réponse automatique.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 11]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
 
-Vous pouvez ensuite cliquer sur `Valider`{.action} pour que le répondeur soit mis en place.
+> [!success]
+>
+> Si vous souhaitez déléguer la gestion d'une réponse automatique pour une adresse e-mail. Consultez notre guide [« Déléguer la gestion de vos comptes e-mails à une autre personne »](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_delegation)
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
> Si vous souhaitez déléguer la gestion d'une réponse automatique pour une adresse e-mail, consultez notre guide [« Déléguer la gestion de vos comptes e-mails à une autre personne »](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_delegation).
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md [hunk 12]
```diff
@@ -39,33 +35,36 @@ La fenêtre d'ajout s'affiche. Vous pouvez la compléter selon les informations
...
 
-### Modification ou supression du répondeur
+### Modification ou suppression d'une réponse automatique
 
-Lorsque votre répondeur e-mail a été créé, il apparaît dans la liste visible dans la section `Gestion des répondeurs`{.action} de votre offre e-mail. Vous pouvez le supprimer ou le modifier en cliquant sur `...`{.action} à droite de celui-ci.
+Lorsque votre réponse automatique a été créée, il apparaît dans la liste visible dans la section `Gestion des répondeurs`{.action} de votre offre e-mail. Vous pouvez le supprimer ou le modifier en cliquant sur `...`{.action} à droite de celui-ci.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/feature_auto_responses/guide.fr-fr.md]:
```suggestion
Lorsque votre réponse automatique a été créée, elle apparaît dans la liste visible dans la section `Gestion des répondeurs`{.action} de votre offre e-mail. Vous pouvez la supprimer ou la modifier en cliquant sur `...`{.action} à droite de celle-ci.
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by BrenKt** [comment]
Translations check to do

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**